forge-gui/res/cardsfolder/upcoming/filigree_vector.txt

```diff
@@ -3,8 +3,10 @@ ManaCost:3 W
 Types:Artifact Creature Phyrexian Construct
 PT:1/1
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerDescription$ When CARDNAME enters the battlefield, put a +1/+1 counter on each of any number of target creatures and a charge counter on each of any number of target artifacts.
-SVar:TrigCounter:DB$ PutCounter | ValidTgts$ Creature | TgtPrompt$ Select any number of target creatures to distribute +1/+1 counters to | CounterType$ P1P1 | CounterNum$ 1 | TargetMin$ 0 | TargetMax$ X | SubAbility$ DBChargeCounter
-SVar:DBChargeCounter:DB$ PutCounter | ValidTgts$ Artifact | TgtPrompt$ Select any number of target artifacts to distribute charge counters to | CounterType$ CHARGE | CounterNum$ 1 | TargetMin$ 0 | TargetMax$ Y
+SVar:TrigCounter:DB$ Pump | ValidTgts$ Creature | TgtPrompt$ Select any number of target creatures to distribute +1/+1 counters to | TargetMin$ 0 | TargetMax$ X | RememberTargets$ True | SubAbility$ DBChargeCounter
+SVar:DBChargeCounter:DB$ Pump | ValidTgts$ Artifact | TgtPrompt$ Select any number of target artifacts to distribute charge counters to | TargetMin$ 0 | TargetMax$ Y | ImprintCards$ ThisTargetedCard | SubAbility$ DBPutCounterAll
```

Comment:
Give this a try–
Use dev mode to give AI a few creatures, an artifact creature, and an artifact.
Then use dev mode to make the AI cast this card...
If the AI doesn't give its stuff counters (or starts giving opponent stuff counters yikes), try adding `AILogic$ Pump` to each of these lines and try again.

Reply:
Yeah AI put a charge counter on my Black Lotus

Reply:
Does the `AILogic$ Pump` seem to make it behave better?

Reply:
AI still giving charge counters to my artifacts 
![image](https://user-images.githubusercontent.com/67333662/235314094-a85f6b3d-f0ce-4bb8-bb49-61b65594989a.png)

Reply:
I guess AI removedeck:Random ?